libobs/obs.h

```diff
@@ -2546,6 +2546,19 @@ static inline void obs_source_frame_destroy(struct obs_source_frame *frame)
 EXPORT void obs_source_frame_copy(struct obs_source_frame *dst,
 				  const struct obs_source_frame *src);
 
+/* ------------------------------------------------------------------------- */
+/* Audio track sources */
+EXPORT obs_source_t *obs_get_audio_track_source(uint32_t track);
+EXPORT int obs_get_audio_track_index(obs_source_t *source);
+EXPORT void obs_set_audio_track_source(uint32_t track, obs_source_t *source);
+EXPORT void obs_check_audio_track_feedback(obs_source_t *source,
+					   bool activating);
+EXPORT bool obs_is_source_audio_track(obs_source_t *source);
+
+EXPORT struct obs_audio_data *
+obs_source_get_output_audio_data(obs_source_t *source,
+				 const struct obs_source_audio *audio);
+
```

Comment:
Documentation is missing about those new functions.

Reply:
I'll do this in a separate PR when this is merged, so we don't make this PR bigger than it already is.

Reply:
I know it's been while. Doing it in another PR makes no sense and is prone to be forgotten to be made.